README.md

```diff
@@ -1,3 +1,16 @@
 # LISTD Media
 
 An app that will allow users to create, share and watch lists of YouTube channels.
+
+## Setup
+###  Getting Google OAuth API Credentials
+
```

Comment:
Add - Move (rename) `.env.example` to `.env`

Reply:
ok i will update 😅